test/e2e/main_test.go

```diff
@@ -33,6 +38,207 @@ var (
 	reportingOperatorImageTag  string
 
 	defaultTargetPods = 7
+
+	reportTestTimeout         = 5 * time.Minute
+	reportTestOutputDirectory string
+
+	testReportsProduceCorrectDataForInputTestCases = []reportsProduceCorrectDataForInputTestCase{
```

Comment:
I think this probably deserves it's own file, and probably next to the testReportsProduceCorrectDataForInput function. Can we add a new file `test/e2e/reports_produce_correct_data_test.go` and move `testReportsProduceCorrectDataForInput` there and `testReportsProduceCorrectDataForInputTestCases` there?